### ISSUE TITLE
fixed-concentration species

### ISSUE THREAD

**Issue by jmuhlich** [pysb/pysb#12]
BNG allows a modeler to declare that the concentration of a species should remain fixed by prefixing it with a $ in the `seed species` block, e.g.  `$EGF(R) 1`.  Maybe implement as a boolean kwarg to Initial, i.e. `Initial(EGF(R=None), 1, fixed=True)`.

**Comment by jmuhlich** [comment]
Note that a workaround for now is to hack your rules so that the fixed species is not consumed. One way to do that is to not actually bind the two monomers and instead use a state site on the downstream monomer to indicate whether it can participate in further reactions:

```python
Rule('L_bind_R_v1', L() + R(ligand='no') <> L() + R(ligand='yes'), kf, kr)
```
Another solution is to produce an extra free copy of the fixed species upon binding:
```python
Rule('L_bind_R_v2', L(r=None) + R(l=None) >> L(r=None) + L(r=1) % R(l=1), kf)
Rule('L_unbind_R_v2', L(r=1) % R(l=1) >> R(l=None), kr)
```
Note the separate reverse reaction here! This is needed because making `L_bind_R_v2` reversible would produce a second order reaction that also depends on the amount of free **L**, which is incorrect. Furthermore, don't write any rules that simply break the L-R bond as that will effectively synthesize extra **L**!

**Comment by alubbock** [comment]
This wouldn't be too difficult to implement, but Kappa doesn't support it AFAIK. Is there much demand for this?

**Comment by alubbock** [comment]
Implemented in #364
